share/functions/__fish_is_git_repository.fish

```diff
@@ -0,0 +1,3 @@
+function __fish_is_git_repository --discription 'Check if current directory is a git repository'
```

Comment:
It's loaded when you request it for the first time. It sources the first `__fish_is_git_repository.fish` it finds the directories listed in `fish_function_path`.

```fish
> echo $fish_function_path 
/home/clement/.config/fish/functions /etc/fish/functions /usr/share/fish/vendor_functions.d /usr/share/fish/functions
```

My guess is that you have a file named `__fish_is_git_repository.fish` without the correction in one of the directories listed before `/usr/share/fish/functions`.